lib/libutils/ext/include/util.h

```diff
@@ -188,7 +188,34 @@ static inline void reg_pair_from_64(uint64_t val, uint32_t *reg0,
 	*reg1 = low32_from_64(val);
 }
 
-/* Get and set bit fields  */
+/**
+ * @brief Functions to get and set bit fields in a 32/64-bit register.
```

Comment:
Please drop the doxygen tags. These comments are read in place and should be optimized for human reading.